src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalThrowsCheck.java

```diff
@@ -84,6 +107,15 @@
  *   &lt;property name=&quot;ignoreOverriddenMethods&quot; value=&quot;false&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *   public void func1() throws Throwable {} // violation
+ * }
+ * class OverrideTest extends Test {
+ *   public void func1() throws RuntimeException {}  // violation
+ * }
```

Comment:
This example is incorrect. Doc says that `allow to ignore checking overridden methods (marked with Override or java.lang.Override annotation).`, I see no annotations